packages/elements/src/react/common/form/index.tsx

```diff
@@ -270,6 +273,29 @@ const useInput = ({
     ref.send({ type: 'FIELD.UPDATE', field: { name, value: initialValue } });
   }, [name, ref, initialValue]);
 
+  const [isSupported, setIsSupported] = React.useState(false);
+  React.useEffect(() => {
+    async function runAutofillPasskey() {
+      const _isSupported = await isWebAuthnAutofillSupported().catch(() => false);
+      setIsSupported(_isSupported);
+    }
+
+    // @ts-expect-error - Depending on type the props can be different
+    if (passthroughProps?.passkeyAutofill) {
+      runAutofillPasskey();
+    }
+
+    // @ts-expect-error - Depending on type the props can be different
+  }, [passthroughProps?.passkeyAutofill]);
```

Comment:
Should this be handled inside an actor ?

Reply:
Which part are you referring to?

Reply:
Yeah, passkey specific logic shouldn't live in the common input element like this. Let's try to move it to an actor.

Why does `passkeyAutofill` need to be on the input?

Reply:
This is only necessary if we want to support [passkey autofill](https://web.dev/articles/passkey-form-autofill#annotate_form_input_field).

Reply:
Should we rely on "the platform" here? It doesn't feel necessary to create a separate abstraction if all it takes is specifying a native property on the element, unless we need to execute some specific logic to enable autofill.